controllers/channels.js

```diff
@@ -277,6 +280,7 @@ module.exports = (app, passport, React, ReactDOMServer) => {
         }
         getPNTokensAndSendPushNotification(members, channelName, message.record.name);
       } catch (e) {
+        console.error('[Error sending messages]', e);
```

Comment:
I have a personal preference with logging the filename or category as the predicate. 
`console.error('[Channels] Error sending message', e)`

Reply:
I fixed the console.log on the feature/logging branch